content/about/news/headless/2019_10_CenterOfExcellence.md

```diff
@@ -1,7 +1,7 @@
 ---
 date: 2019-10-01
-title: Delphi named 'Center of Excellence'
+title: Center of Excellence for Influenza Forecasting
```

Comment:
Style note -- all the other news items are titled with complete sentences: subject-verb-object. This is just a noun. Keep this change or be consistent with the other news items?